modules/form/src/addons/scopes.tsx

```diff
@@ -0,0 +1,249 @@
+/**
+ * Copyright (c) 2021, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import filter from "lodash-es/filter";
+import isEmpty from "lodash-es/isEmpty";
+import isEqual from "lodash-es/isEqual";
+import React, { FunctionComponent, ReactElement, useEffect, useState } from "react";
+import { Button, Form, Icon, Label, Message, Popup } from "semantic-ui-react";
+
+interface ScopesPropsInterface {
+    defaultValue: string;
```

Comment:
Explain the interface properties.
```suggestion
    defaultValue: string;
```

Reply:
fixed with 51ded5bd3ca56d4ec2c9f85a6b73b30573767c72